Sources/TuistKit/Services/GraphService.swift

```diff
@@ -11,6 +11,10 @@ import TuistSupport
 final class GraphService {
     private let graphVizMapper: GraphToGraphVizMapping
     private let manifestGraphLoader: ManifestGraphLoading
+    
+    private enum GraphServiceError: Error {
```

Comment:
To be consistent with the rest of the codebase, I'd recommend to conform the `FatalError` protocol, and use cases to represent the error type. `FatalError` requires errors to include a description, which is presented automatically by an error handler that catches fatal errors when they bubble up, and the type is used to determine whether the error should get automatically reported.


```swift
private enum GraphServiceError: FatalError {
  case invalidFormat(String)

  var description: String {
    switch self {
      case .invalidFormat(let format): "....."
    }
  }

  var type: ErrorType {
    switch self {
      case .invalidFormat: return .abort
    }
  }
}
```

Reply:
Thank you for the suggestion. Let's do that.